src/m365/flow/commands/recyclebinitem/recyclebinitem-list.spec.ts

```diff
@@ -0,0 +1,253 @@
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name, commands.RECYCLEBINITEM_LIST);
+  });
+
+  it('has a description', () => {
+    assert.notStrictEqual(command.description, null);
+  });
+
+  it('defines correct default properties', () => {
+    assert.deepStrictEqual(command.defaultProperties(), ['name', 'displayName']);
+  });
+
+  it('outputs exactly one result when retrieving deleted flows with output json', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://api.flow.microsoft.com/providers/Microsoft.ProcessSimple/scopes/admin/environments/${formatting.encodeQueryParameter(environmentName)}/v2/flows?api-version=2016-11-01&include=softDeletedFlows`) {
+        return flowResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, environmentName: environmentName } });
+    assert(loggerLogSpy.calledOnce);
+  });
+
+  it('outputs exactly one result when retrieving deleted flows with output text', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://api.flow.microsoft.com/providers/Microsoft.ProcessSimple/scopes/admin/environments/${formatting.encodeQueryParameter(environmentName)}/v2/flows?api-version=2016-11-01&include=softDeletedFlows`) {
+        return flowResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, environmentName: environmentName, output: 'text' } });
+    assert(loggerLogSpy.calledOnce);
+  });
+
+  it('correctly retrieves deleted flows with output json', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://api.flow.microsoft.com/providers/Microsoft.ProcessSimple/scopes/admin/environments/${formatting.encodeQueryParameter(environmentName)}/v2/flows?api-version=2016-11-01&include=softDeletedFlows`) {
+        return flowResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, environmentName: environmentName } });
+    assert.deepStrictEqual(loggerLogSpy.firstCall.args[0], deletedFlows);
+  });
+
+  it('correctly retrieves deleted flows with output text', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://api.flow.microsoft.com/providers/Microsoft.ProcessSimple/scopes/admin/environments/${formatting.encodeQueryParameter(environmentName)}/v2/flows?api-version=2016-11-01&include=softDeletedFlows`) {
+        return flowResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, environmentName: environmentName, output: 'text' } });
+    const textResponse = deletedFlows.map(flow => ({ ...flow, displayName: flow.properties.displayName }));
+
+    assert.deepStrictEqual(loggerLogSpy.firstCall.args[0], textResponse);
+  });
+
```

Comment:
Couldn't we combine this assert with the first text test? Now we are doing the exact same

Reply:
You want to merge 2 tests? They are doing the same indeed but testing something different. AFAIK, in an ideal world, a test should have only 1 assert, so it checks exactly 1 thing.